docs/man-openmpi/man3/MPI_Comm_spawn_multiple.3.rst

```diff
@@ -251,7 +251,7 @@ Other restrictions apply to the *array_of_argv* parameter; see
 :ref:`MPI_Comm_spawn`'s description of the *argv* parameter for more
 details.
 
-MPI-3.1 implies (but does not directly state) that the argument
+MPI-|mpi_standard_version|.|mpi_standard_subversion| implies (but does not directly state) that the argument
```

Comment:
Didn't this get fixed in MPI-4.0?  I.e., when we start stating that Open MPI supports MPI-4.0, I think that this sentence may no longer be true...?  (that being said, don't trust my memory: this would need to be checked against the MPI-4.0 text)

Reply:
For reference, v4.0 and v3.1 say the following:

```
IN count number of commands (positive integer, significant to
MPI only at root — see advice to users)

IN array_of_commands programs to be executed (array of strings, significant
only at root)
...
MPI_COMM_SPAWN_MULTIPLE is identical to MPI_COMM_SPAWN except that there
are multiple executable specifications. The first argument, count, gives the number of
specifications.
```
To me it seems pretty clear that count should be the length of `commands`, in v3.1 and v4.0. :man_shrugging:

Reply:
Ok, I could have sworn that there was some discussion about this at the Forum level, but then again, it's been quite a while since I've been to a Forum meeting...